front/temporal/labs/client.ts

```diff
@@ -21,7 +21,7 @@ export async function launchRetrieveTranscriptsWorkflow(
       args: [transcriptsConfiguration.id],
       taskQueue: QUEUE_NAME,
       workflowId: workflowId,
-      cronSchedule: "*/5 * * * *",
+      // cronSchedule: "*/5 * * * *",
```

Comment:
this on purpose ?

Reply:
Nah shouldn't have been committed -reverting